src/demo/java/org/junitpioneer/jupiter/cartesian/CartesianTestExtensionDemo.java

```diff
@@ -282,5 +284,13 @@ ArgumentSets provideArguments() {
 	}
 	// end::cartesian_argument_sets_with_non_static_factory[]
 
+	static class MyTestReporter implements TestReporter {
```

Comment:
What is this for?

Reply:
This is for the documentation - for demonstrating an edge case with CartesianTest